Framework/API/inc/MantidAPI/ComponentInfo.h

```diff
@@ -63,12 +63,10 @@ class MANTID_API_DLL ComponentInfo {
           componentIds,
       boost::shared_ptr<const std::unordered_map<
           Geometry::IComponent *, size_t>> componentIdToIndexMap);
-  std::vector<size_t> detectorIndices(size_t componentIndex) const;
-  std::vector<size_t> componentIndices(size_t componentIndex) const;
+  std::vector<size_t> detectorsInSubTree(size_t componentIndex) const;
+  std::vector<size_t> componentsInSubTree(size_t componentIndex) const;
```

Comment:
This is nitpicking, but maybe consider "subtree" as one word, i.e., use `detectorInSubtree`?